core/src/message/context.rs

```diff
@@ -141,34 +141,40 @@ impl ContextOutcome {
         }
     }
 }
+/// Store of current temporary message execution context.
+#[derive(Clone, Default, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, Decode, Encode, TypeInfo)]
+pub struct OutgoingPayloads {
+    outgoing: BTreeMap<u32, Option<Payload>>,
```

Comment:
I'd rename for clearance 
```suggestion
    handle: BTreeMap<u32, Option<Payload>>,
```